codec_context.go

```diff
@@ -314,6 +314,14 @@ func (cc *CodecContext) SetHardwareDeviceContext(hdc *HardwareDeviceContext) {
 	}
 }
 
+func (cc *CodecContext) ExtraHwFrames() int {
```

Comment:
Could you rename to `ExtraHardwareFrames`?

Reply:
Done